app/lab/package.json

```diff
@@ -232,6 +232,7 @@
       "@jupyterlab/tooltip",
       "@jupyterlab/translation",
       "@jupyterlab/ui-components",
+      "@jupyterlite/contents",
```

Comment:
:+1: on putting in lexicographical order :mage:

Reply:
Ah, looks like it needs to appear in:

```
Checking app `resolutions`...
... app/lab/package.json `resolutions` are out-of-date!
... app/repl/package.json `resolutions` are up-to-date!
... app/retro/package.json `resolutions` are up-to-date!
```

The `resolutions` field helps keep the installed versions working properly... not so relevant for lite, as of yet, but kind of a big deal with jupyterlab, which has multiple active "head" versions (e.g. 2.x, 3.5.x, 3.6.x, 4.x).

Reply:
ahh ok thanks for catching that. I'm pretty new to the "under the hood" part of how the jlab system works. Thanks for the patience

Reply:
yeah: this is kinda like working at the engine fab rather than the auto mechanic's shop, trying to figure out how to make engines that work in more kinds of cars.

Of note, there are a few other custom jupyterlab apps out there that can make use of essentially "vanilla" labextensions (jupyterlite adds the "serverextension" flavor, so is decidedly non-"standard"):
- https://github.com/voila-dashboards/voila/tree/main/packages/voila
- https://github.com/jupyter/notebook/tree/main/packages/application